java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture7/tasks/_2/CalculateAtomicObjects.java

```diff
@@ -0,0 +1,57 @@
+package com.zoolatech.lecture7.tasks._2;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class CalculateAtomicObjects implements Operations {
+    private final AtomicInteger getValue;
+    private int value;
+
+    public CalculateAtomicObjects(int value, AtomicInteger getValue) {
+        this.value = value;
+        this.getValue = getValue;
+    }
+
+    public AtomicInteger currentValueForAtomicInteger() {
```

Comment:
method should return int value - AtomicInteger is an implementation detail, and should be hidden inside of the class - nobody outside needs to know about it